Remotion/ObjectBinding/Web.UnitTests/UI/Controls/Validation/BocListValidationResultDispatchingValidatorTest.cs

```diff
@@ -145,5 +143,125 @@ public void LazyValidate_WithInternalAndRowFailures_ReturnsAppendedMessageText (
 
       Assert.That(validator.ErrorMessage, Is.EqualTo("Errors on BocList\r\nErrors on rows in BocList"));
     }
+
+    [Test]
+    public void EvaluateIsValid_WithoutFailures_EmptyErrormessageAndValid ()
+    {
+      var validationFailureRepoStub = new Mock<IBocListValidationFailureRepository>();
+      validationFailureRepoStub.Setup(_ => _.HasValidationFailuresForDataRow(It.IsAny<IBusinessObject>())).Returns(false);
```

Comment:
The simple rule that was employed in the past  is the following: if the value is "important" to setting up the test, we should still stub it so the behavior/usage is documented. if it's just guard condition satisfaction that's not relevant for the test, we can leave it out. 

So, if we test: 
* do we get "validation failures found" when a validator returns "false" when calling `Validate()`, we should have two tests, one where `Validate()` returns true and one where it returns false. 
* Same goes for a test that actually is intended to verify that an exit condtion is triggered upon false. 

All other cases where the default-return-value case is already covered and noone cares about this check, we can skip stubbing it. 

As for this particular case: If that's the only / primary setup of the test, we should stub it. From the name of the test but without looking at the rest of the code, it might really do feel like it's relevant information that this method causes the emtpy result. Similiar situation would exist for instance when testing a reader-api on a stream and we want to test when the stream is empty. we would still stub reader to return 0 when calling the read method. Otherwise, we have a mock, do nothing with it and then get a result. that just feels...weird when reading the test.

Also, casing-typo in the Test name ("Errormessage" -> "ErrorMessage")

@a-ctor